nasl/nasl_packet_forgery_v6.c

```diff
@@ -1688,7 +1689,26 @@ nasl_send_v6packet (lex_ctxt *lexic)
       bzero (&sockaddr, sizeof (struct sockaddr_in6));
       sockaddr.sin6_family = AF_INET6;
       sockaddr.sin6_addr = sip->ip6_dst;
-      if (dstip != NULL && !IN6_ARE_ADDR_EQUAL (&sockaddr.sin6_addr, dstip))
+
+      if (allow_multicast)
+        {
+          struct sockaddr_in6 multicast;
+
+          if (setsockopt (soc, SOL_SOCKET, SO_BROADCAST, &offset,
```

Comment:
Similar to nasl_send_packet, this is not an offset, but an on/off optval, so the variable naming is slightly confusing.